GitUI/Script/ScriptOptionsParser.cs

```diff
@@ -52,31 +55,41 @@ public sealed class ScriptOptionsParser
             "WorkingDir"
         };
 
-        [NotNull]
-        private static string CreateOption([NotNull] string option, bool quoted)
-        {
-            var result = "{" + option + "}";
-
-            if (quoted)
-            {
-                result = "{" + result + "}";
-            }
-
-            return result;
-        }
-
+        /// <summary>
+        ///  Checks whether <paramref name="arguments"/> contains the supplied <paramref name="option"/>.
+        /// </summary>
+        /// <param name="arguments">The script argument.</param>
+        /// <param name="option">The option.</param>
+        /// <returns><see langword="true"/> if the argument contains the option; otherwise <see langword="false"/>.</returns>
         public static bool Contains([NotNull] string arguments, [NotNull] string option)
         {
+            arguments = arguments ?? throw new ArgumentNullException(nameof(arguments));
+            option = option ?? throw new ArgumentNullException(nameof(option));
+
             return arguments.Contains(CreateOption(option, quoted: false));
         }
 
+        /// <summary>
+        ///  Checks whether <paramref name="option"/> starts with "s".
+        /// </summary>
+        /// <param name="option">The option.</param>
+        /// <returns><see langword="true"/> if the options starts with "s"; otherwise <see langword="false"/>.</returns>
         public static bool DependsOnSelectedRevision([NotNull] string option)
         {
+            option = option ?? throw new ArgumentNullException(nameof(option));
+
             return option.StartsWith("s");
         }
 
-        public static (string arguments, bool abort) Parse(string arguments, IGitModule module, IWin32Window owner, RevisionGridControl revisionGrid)
+        public static (string arguments, bool abort) Parse([CanBeNull] string arguments, [NotNull] IGitModule module, IWin32Window owner, IScriptHostControl scriptHostControl)
         {
+            if (string.IsNullOrWhiteSpace(arguments))
+            {
+                return (arguments, abort: false);
+            }
+
+            module = module ?? throw new ArgumentNullException(nameof(module));
```

Comment:
```suggestion
            module ??= throw new ArgumentNullException(nameof(module));
```

Reply:
This actually is breaking the build...
![image](https://user-images.githubusercontent.com/4403806/79068690-2316d580-7d0c-11ea-9063-41fb7f0190db.png)